src/todo/UI.lua

```diff
@@ -195,6 +195,30 @@ function todo.create_add_edit_frame(player, task)
         selected_index = assign_index
     })
 
+    if (task) then
+        table.add({
+            type = "label",
+            style = "todo_label_default",
+            name = "todo_by_label",
+            caption = "Created/modified by"
```

Comment:
Should this be localised?

Reply:
Yes, I just totally forgot to set this right...